docs/contribute/maintenance.rst

```diff
@@ -29,6 +29,8 @@ Maintainers need the following permissions.
     This Google Group is used for managing `Code of Conduct <https://github.com/collective/icalendar/blob/main/CODE_OF_CONDUCT.md>`_ infringement reports.
     ``Manager``\ s may manage and moderate messages, whereas ``Owner``\ s may also manage members.
     Management is performed through `Google Groups icalendar-coc settings <https://groups.google.com/g/icalendar-coc/settings>`_.
+``INSERT_ROLE_HERE`` access to the `OSS Fuzz issue tracker <https://issues.oss-fuzz.com/u/5/issues?q=icalendar>`_ for icalendar.
```

Comment:
@niccokunzmann please replace `INSERT_ROLE_HERE` with the actual role. I don't have private access.

Reply:
b355d74